test/libscap/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ set(LIBSCAP_TESTS_DEPENDENCIES
 file(GLOB_RECURSE GENERIC_TEST_SUITE "${CMAKE_CURRENT_SOURCE_DIR}/test_suites/generic/*.cpp")
 set(LIBSCAP_TESTS_SOURCES ${GENERIC_TEST_SUITE})
 
+# Test suite asserting against tables
+file(GLOB_RECURSE TABLES_TEST_SUITE "${CMAKE_CURRENT_SOURCE_DIR}/test_suites/tables/*.cpp")
+set(LIBSCAP_TESTS_SOURCES ${TABLES_TEST_SUITE})
+
```

Comment:
I created a new test suite in scap to assert against all our tables. Right now we assert our table in sinsp but this is not the right place IMHO! A follow-up PR should bring some sinsp tests into this scap suite. Moreover, right now we have 2 test folders for scap, one inside `test/libscap` (old one) and one inside `test` (new-one) in this follow-up PR we should also remove the old folder since all tests are already ported inside the new one